offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -369,6 +398,81 @@ export class CkbHandler {
     }
   }
 
+  async onLockTx(txInfo: CkbTxInfo, currentHeight: number): Promise<void> {
+    const tx = txInfo.tx.transaction;
+    const txHash = txInfo.info.tx_hash;
+    const records = await this.db.getCkbLockByTxHashes([txHash]);
+    if (records.length > 1) {
+      logger.error('unexpected db find error', records);
+      throw new Error(`unexpected db find error, records.length = ${records.length}`);
+    }
+    const cellData = await this.parseLockTx(tx);
```

Comment:
better verify lockTx inside `fn parseLockTx`, if the tx is a validate lockTx, then return parsed NervosLockAssetTxMetaData, otherwise return null

Reply:
added some verify logic inside parseLockTx